pkg/services/secrets/manager/manager.go

```diff
@@ -56,6 +57,10 @@ type dataKeyCacheItem struct {
 var b64 = base64.RawStdEncoding
 
 func (s *SecretsService) Encrypt(ctx context.Context, payload []byte, opt secrets.EncryptionOptions) ([]byte, error) {
+	return s.EncryptWithDBSession(ctx, payload, opt, nil)
+}
+
+func (s *SecretsService) EncryptWithDBSession(ctx context.Context, payload []byte, opt secrets.EncryptionOptions, sess *sqlstore.DBSession) ([]byte, error) {
```

Comment:
This will result into cyclic dependency as well
```
github.com/grafana/grafana/pkg/services/ngalert/notifier/channels
package github.com/grafana/grafana/pkg/services/ngalert/notifier/channels
        imports github.com/grafana/grafana/pkg/services/sqlstore/migrations/ualert
        imports github.com/grafana/grafana/pkg/services/sqlstore/migrations
        imports github.com/grafana/grafana/pkg/services/sqlstore
        imports github.com/grafana/grafana/pkg/services/secrets/manager: import cycle not allowed in test
FAIL    github.com/grafana/grafana/pkg/services/ngalert/notifier/channels [setup failed]
```